docarray/documents/text.py

```diff
@@ -102,10 +104,22 @@ class MultiModalDoc(BaseDoc):
 
     """
 
-    text: Optional[str]
-    url: Optional[TextUrl]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[bytes]
+    text: Optional[str] = Field(
+        description='The text content stored in the document',
+        example='This is an example text content of the document',
+    )
+    url: Optional[TextUrl] = Field(
+        description='''URL to a (potentially remote) text file that needs to be
+        loaded''',
```

Comment:
```suggestion
        description='''URL to a (potentially remote) text file that can be
        loaded''',
```
Who are we to tell our users what to do ;)